java-checks/src/main/resources/org/sonar/l10n/java/rules/java/S6814.json

```diff
@@ -0,0 +1,23 @@
+{
+  "title": "Optional REST parameters should have an object type",
+  "type": "CODE_SMELL",
+  "status": "ready",
+  "remediation": {
+    "func": "Constant\/Issue",
+    "constantCost": "5min"
+  },
+  "tags": [],
+  "defaultSeverity": "Major",
+  "ruleSpecification": "RSPEC-6814",
+  "sqKey": "S6814",
+  "scope": "All",
+  "quickfix": "unknown",
+  "code": {
+    "impacts": {
+      "MAINTAINABILITY": "HIGH",
+      "RELIABILITY": "MEDIUM",
+      "SECURITY": "LOW"
```

Comment:
Do we expect that this has any impact at all on e.g. security? If yes, ignore this comment, if now, should we remove the security key?

Reply:
Indeed, I will remove that